manifests/init.pp

```diff
@@ -183,9 +134,33 @@
     '/usr/share/containers/oci/hooks.d': ;
   }
 
+  if $manage_cilium {
+    if fact('os.name') == 'Ubuntu' and fact('os.release.full') == '22.04' {
```

Comment:
If this is something related to systemd-resolved, then it should probably check for that instead of doing assumptions based on the OS name/version

Reply:
yes this could be better to do it this way around. will patch this.